projects/packages/my-jetpack/_inc/components/plans-section/style.module.scss

```diff
@@ -8,10 +8,9 @@
 	margin-bottom: calc( var(--spacing-base ) * 3 );
 }
 
-.external-link {
-	font-size: var( --font-body );
-	&:hover {
-		color: var( --jp-black );
-		text-decoration-thickness: var( --jp-underline-thickness );
+.actions-list {
+	li {
```

Comment:
Since you're styling only `li`, suggest renaming the class and applying directly at `li`.